volume/store/store.go

```diff
@@ -120,26 +120,8 @@ func (s *VolumeStore) List() ([]volume.Volume, []string, error) {
 	if err != nil {
 		return nil, nil, &OpErr{Err: err, Op: "list"}
 	}
-	var out []volume.Volume
 
-	for _, v := range vols {
-		name := normaliseVolumeName(v.Name())
-
-		s.locks.Lock(name)
-		storedV, exists := s.getNamed(name)
-		if !exists {
-			s.setNamed(v, "")
-		}
-		if exists && storedV.DriverName() != v.DriverName() {
```

Comment:
Yes, we strive to make sure they don't exist, since there is no way to actually pick 1 volume or the other